routes/login.js

```diff
@@ -0,0 +1,31 @@
+const { query } = require("../database/connection")
+const login = require("express").Router()
+const bcrypt = require("bcrypt")
+
+login.post("/", async (req, res) => {
+  try {
+    let userInfo = await query(`SELECT * FROM users WHERE email = '${req.body.email}';`);
+
```

Comment:
Take everything from this line, all the way to line 25, basically the if-else within the try, and place it after the try catch. Basically, unnest the stuff inside; make your code call '_query_' in the _try_ brackets, and then jump immediately to the spot right after _catch_ brackets and run all the other stuff. Like try doing it like how I did it in my register file.